docs/source/markdown/podman-stats.1.md

```diff
@@ -30,17 +30,35 @@ Pretty-print container statistics to JSON or using a Go template
 
 Valid placeholders for the Go template are listed below:
 
-| **Placeholder** | **Description**    |
-| --------------- | ------------------ |
-| .ID             | Container ID       |
-| .Name           | Container Name     |
-| .CPUPerc        | CPU percentage     |
-| .MemUsage       | Memory usage       |
-| .MemUsageBytes  | Memory usage (IEC) |
-| .MemPerc        | Memory percentage  |
-| .NetIO          | Network IO         |
-| .BlockIO        | Block IO           |
-| .PIDS           | Number of PIDs     |
+| **Placeholder**     | **Description**                                          |
+|---------------------|----------------------------------------------------------|
+| .ID                 | Container ID (truncated)                                 |
+| .ContainerID        | Container ID (full hash)                                 |
+| .Name               | Container Name                                           |
+| .CPUPerc            | CPU percentage                                           |
+| .MemUsage           | Memory usage                                             |
+| .MemUsageBytes      | Memory usage (IEC)                                       |
+| .AvgCPU             | Average CPU (full precision float)                       |
+| .AVGCPU             | Average CPU (formatted)                                  |
+| .CPU                | Percent CPU (full precision float)                       |
+| .CPUNano            | CPU Usage, total, in nanoseconds                         |
+| .CPUSystemNano      | CPU Usage, kernel, in nanoseconds                        |
+| .Duration           | Same as CPUNano (FIXME: why?)                            |
+| .MemLimit           | Memory limit, in bytes                                   |
+| .MemPerc            | Memory percentage                                        |
+| .NetIO              | Network IO                                               |
+| .BlockIO            | Block IO                                                 |
+| .PIDS               | Number of PIDs                                           |
+| .NetInput           | Network Input                                            |
+| .NetOutput          | Network Output                                           |
+| .BlockInput         | Block Input                                              |
+| .BlockOutput        | Block Output                                             |
+| .PIDs               | Number of PIDs                                           |
+| .PerCPU             | FIXME someone help please                                |
+| .ContainerStats ... | WTF WTF WTF 2022-09-12 this is a total dup dup?          |
+| .SystemNano         | Current system datetime, nanoseconds since epoch         |
+| .UpTime             | WTF? This is Duration (CPUNano) in human-readable form?? |
```

Comment:
If possible, it would be nice to have this list in alpha order.

Reply:
Oh boy, that one hasn't cost me sleep, but has cost me a lot of wasted time thinking about it. I settled on alpha-ish, in which important fields (ID, Name) are at top, and identical ones (ID & ContainerID, CreateCgroup & CgroupParent) are next to each other. But I'm glad you started the discussion, and welcome thoughts on what the least-worst approach is.

Reply:
I would guess the ContainerStats might be the problem.

@edsantiago You could break this into default ones and then additional ones.

Reply:
Like, two tables, one "You probably want..." and one "...but there's also"? That's doable, and seems user-friendly to me. I don't really have the judgment to know what's what, though.

And, thinking about it, now that I know how to put comments in a `.md` file ... if it's absolutely necessary to have fields that are spit out by command-completion but which must not be documented, we could (maybe) hide those fields in the markdown, in a way that the xref script will see them but human readers of the man page won't. (Users would still see those when they hit TAB though; they just wouldn't be in the man page). I don't especially care for this idea, but sometimes lesser evil etc.